thorlcr/thorutil/thmem.cpp

```diff
@@ -1720,11 +1720,20 @@ class CThorRowCollectorBase : public CSpillable
                 oom = true;
             if (oom)
             {
-                ReleaseThorRow(row);
+                if (_release)
+                    ReleaseThorRow(row);
                 throw MakeActivityException(&activity, ROXIEMM_MEMORY_LIMIT_EXCEEDED, "Insufficient memory to append sort row");
             }
         }
     }
+    void putRow(const void *row)
+    {
+        _putRow(row, true);
+    }
+    void writeRow(const void *row)
+    {
+        _putRow(row, false);
```

Comment:
This is not leagal - spillableRows.append is taking ownership

Reply:
I missed that; thanks. Corrected.